tests/test_config.py

```diff
@@ -9,6 +9,7 @@
 from autogpt.configurator import create_config
 
 
+# import
```

Comment:
?

Reply:
This is the result from running `isort` and/or `black` linters